editorial-communication/cover-letter-submission.Rmd

```diff
@@ -11,24 +11,24 @@ output:
-Formal academic courses on these types of tools in EEB are often unavailable and researchers are left to learn through informal blogs, videos, and other online forums. 
+Formal academic courses on these types of tools are often unavailable and researchers are left to learn through informal blogs, videos, and other online forums. 
 This may present a plethora of practical barriers to applying adequate standards to maintaining scientific code.
 
+The adoption of GitHub in ecology and evolutionary biology workflows is not widespread in part because there have been only a few disparate attempts at creating domain-specific information and guidelines.
+In our study, we review and discuss one of the most used web-based platforms for computational version control and collaboration, GitHub, and provide researchers in ecology and evolution with comprehensive and practical workflows aimed at facilitating their scientific code and management process.
 
-The adoption of GitHub in EEB workflows is not widespread in part because there have been only a few disparate attempts at creating domain-specific information and guidelines.
-In our study, we review and discuss one of the most used web-based platforms for computational version control and collaboration, GitHub, and provide researchers in EEB with comprehensive and practical workflows aimed at facilitating their scientific code and management process.
-
-Our manuscript is the most accessible and practical guide to using GitHub in EEB research to date. 
-We build upon some existing resources, which are mainly targeted at other disciplines, and our own experiences (from across many EEB subdisciplines) in writing and sharing code openly, to highlight steps EEB researchers can take to improve their research practices in a reproducible, collaborative, and transparent way.
-Our manuscript is important for the *Nature Ecology & Evolution* readership because it will give EEB researchers key resources and workflows to guide them through migrating workflows to GitHub and supporting more open and collaborative science.
+Our manuscript is the most accessible and practical guide to using GitHub in ecology and evolution research to date.
+We build upon our own experiences (as an international collaboration from authors across many ecology and evolution subdisciplines) and on existing resources mainly targeted at other disciplines, to highlight how researchers can use the readily available GitHub suite to make their research practices reproducible, collaborative, and transparent.
+Our manuscript is important for the *Nature Ecology & Evolution* readership because researchers in ecology and evolutionary biology can greatly benefit from our guide to migrate and incorporate GitHub workflows in their research practices ultimately towards an open and collaborative science.
 
 The enclosed manuscript is available as a preprint on the NAME OF SERVER AND CITATION, but is not published or in review elsewhere.
```

Comment:
A general question (@robcrystalornelas) should we preprint earlier than we submit or simultaniously - I'm thinking if we do it a few weeks beforehand we can push the preprint on twitter and elsewhere and then in the cover letter can say something along the lines of " our preprint has been downloaded XXX times in two weeks illustrating how important it is for our community"... just a thought.